php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/PHPVisitorCheck.java

```diff
@@ -520,6 +521,7 @@ protected void scan(Tree tree) {
     }
   }
 
+  // fixme (@Lena) : test this test. Looks like it visits only children of test elements
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1134)